src/UserInputHandler.js

```diff
@@ -0,0 +1,85 @@
+import { Console } from '@woowacourse/mission-utils';
+import {
+  CONSOLE_MESSAGE,
+  ERROR_MESSAGE,
+  MAX_CAR_COUNT,
+  MAX_TRY_COUNT,
+} from './constant.js';
+import {
+  errorString,
+  isInputEmpty,
+  isIntegerNumber,
+  isNumber,
+  isPositiveNumber,
+} from './util.js';
+
+class UserInputHandler {
+  async getCarNames() {
+    try {
+      const carInput = await this.#getUserInput(
+        CONSOLE_MESSAGE.CAR_INPUT_MESSAGE,
+      );
+      this.#validateCarInput(carInput);
+
+      return carInput.split(',').map((carName) => carName.trim());
+    } catch (error) {
+      throw new Error(error.message);
+    }
+  }
+
+  async getTryCount() {
+    try {
+      const tryCountInput = await this.#getUserInput(
+        CONSOLE_MESSAGE.TRY_COUNT_INPUT_MESSAGE,
+      );
+      this.#validateTryCountInput(tryCountInput);
+
+      return Number(tryCountInput);
+    } catch (error) {
+      throw new Error(error.message);
+    }
+  }
+
+  #getUserInput = async (message) => {
+    const input = await Console.readLineAsync(message);
+    if (!input || isInputEmpty(input)) {
+      throw new Error(errorString(ERROR_MESSAGE.EMPTY_INPUT));
+    }
+
+    return input;
+  };
+
+  #validateCarInput(input) {
+    if (!input.includes(',')) {
+      throw new Error(errorString(ERROR_MESSAGE.MIN_CAR_COUNT));
+    }
+
+    const carNames = input.split(',').map((carName) => carName.trim());
```

Comment:
이 로직이 `getCarNames()` 내부에도 있는 것으로 보이는데 util이나 함수로 분리하는 것은 어떨까요? 불필요할까요?
```suggestion
    function inputToCarNames(input) {
      return input.split(',').map((carName) => carName.trim());
    }
    const carNames = inputToCarNames(input);
```

이런식으로요! 단순한 제안이니 참고만 해주세용 ㅎㅎ